Resync `css/CSS2/tables` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 0217df2d1a8a09bb5f2639a13307b789434566a0
<pre>
Resync `css/CSS2/tables` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=301812">https://bugs.webkit.org/show_bug.cgi?id=301812</a>
<a href="https://rdar.apple.com/163866306">rdar://163866306</a>

Reviewed by Matthieu Dubet and Brandon Stewart.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/bc9587088f1e5621317f3db04b0f0d5f8e008b4f">https://github.com/web-platform-tests/wpt/commit/bc9587088f1e5621317f3db04b0f0d5f8e008b4f</a>

* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/tables/border-spacing-095-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/tables/border-spacing-095.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/tables/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/302443@main">https://commits.webkit.org/302443@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/80b54a478ea9f31f596d720d5098abff45f6d48b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/129106 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/1365 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/39942 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/136485 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/80482 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/8b843fdc-1ee8-454a-9dc4-a07739edc129) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/1297 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/1242 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/98302 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/66175 "") | [❌ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/4e941c31-4208-42d7-878a-27b07521c86e) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/132053 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/1006 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/115650 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/78947 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/4ff74b1b-257e-4506-8add-a330198761dd) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/930 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/33767 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/79764 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/109377 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/34263 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/138959 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/1158 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/1127 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/106838 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/1210 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/111986 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/106665 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/27153 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/952 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/30510 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/53716 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/1231 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/1057 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/1103 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/1155 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

stwrt reviewed (approved):
Should this be mentioning importing an expected file in the import log?

Ahmad-S792 committed:
Resync `css/CSS2/tables` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=301812
rdar://163866306

Reviewed by Matthieu Dubet and Brandon Stewart.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/bc9587088f1e5621317f3db04b0f0d5f8e008b4f

* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/tables/border-spacing-095-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/tables/border-spacing-095.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/tables/w3c-import.log:

Canonical link: https://commits.webkit.org/302443@main
